src/actions/portfolio/handlers/bitcoin/index.js

```diff
@@ -0,0 +1,25 @@
+import { getInfoFromCoinGecko, getBtcFromSatoshis } from "../../../../utils";
+import Decimal from "decimal.js";
+
+export const getBitcoinPortfolio = async (
+    address,
+    fiatCurrency,
+    coinGeckoIds
+) => {
+    const response = await fetch(
+        `https://blockchain.info/rawaddr/${address}?limit=0`
+    );
+    if (!response.ok) {
+        throw new Error("invalid response");
+    }
+    const json = await response.json();
+    const portfolio = [];
+    const { final_balance } = json;
+
+    portfolio.push({
```

Comment:
To be more succint, you could do 

```
return [{
    symbol: "BTC",
    balance: getBtcFromSatoshis(new Decimal(final_balance)),
    info: await getInfoFromCoinGecko(coinGeckoIds.btc, fiatCurrency),
}]
```
Seeing a variable name in snake-case also kinda hurts my eyes. What about doing:

`const { final_balance: finalBalance } = json;`

?